winterize/mod.conf

```
@@ -0,0 +1,2 @@
+name = winterize
+depends = default, ctf_map, wool
```

The mod is `ctf_map_core` since a recent commit.
```suggestion
depends = default, ctf_map_core, wool
```